docs/script-api.md

```diff
@@ -225,7 +225,7 @@ Add a button that the user can click in the UI. Sets the timestamp of last click
 Spawn a "block" of liquids, always spawns in the front layer and will have fun effects if `entity_type` is not a liquid.
 Don't overuse this, you are still restricted by the liquid pool sizes and thus might crash the game.
 ### [`spawn_entity`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=spawn_entity)
-`int spawn_entity(int entity_type, float x, float y, int layer, float vx, float vy)`<br/>
+`int spawn_entity(int entity_type, float x, float y, int enum_layer, float vx, float vy)`<br/>
```

Comment:
Instead of renaming them all maybe add a typedef for it and specify the parameter as that. See for example what we do with `VANILLA_SOUND`